test/commands/instance/spawn.test.ts

```diff
@@ -2,13 +2,15 @@ import { expect, test } from '@oclif/test'
 
 describe('instance:spawn', () => {
   test
+    .timeout(10000)
```

Comment:
shall we add https://eslint.org/docs/rules/no-magic-numbers ?

Reply:
See my answer below